compiler/compile/OMRNonHelperSymbols.enum

```diff
@@ -365,7 +365,12 @@
     */
    j9VMThreadFloatTemp1Symbol,
 
-   OMRlastPrintableCommonNonhelperSymbol = j9VMThreadFloatTemp1Symbol,
+   /** \brief
+    * This symbol represents the vTableIndex field in struct J9JNIMethodID
+    */
+   J9JNIMethodIDvTableIndexFieldSymbol
```

Comment:
Is there a comma missing at the end of this line?

Reply:
Fixed.